From 824f30e3a045347806deedb75f16e96b057f430b Mon Sep 17 00:00:00 2001
From: Vaibhav Bedia <vaibhav.bedia@ti.com>
Date: Tue, 22 May 2012 16:25:53 +0530
Subject: [PATCH 01/18] ARM: OMAP: AM33XX: Add missing EMIF register offsets

Add a couple of missing register offsets in the EMIF
header file. While here also fix the indentation issue.

Signed-off-by: Vaibhav Bedia <vaibhav.bedia@ti.com>
---
 arch/arm/plat-omap/include/plat/emif.h |   34 ++++++++++++++++----------------
 1 files changed, 17 insertions(+), 17 deletions(-)

diff --git a/arch/arm/plat-omap/include/plat/emif.h b/arch/arm/plat-omap/include/plat/emif.h
index 314c126..445cbb1 100644
--- a/arch/arm/plat-omap/include/plat/emif.h
+++ b/arch/arm/plat-omap/include/plat/emif.h
@@ -17,23 +17,23 @@
 #define __EMIF_H
 
 #define EMIF_MOD_ID_REV			(0x0)
-#define EMIF4_0_SDRAM_STATUS            (0x04)
-#define EMIF4_0_SDRAM_CONFIG            (0x08)
-#define EMIF4_0_SDRAM_CONFIG2           (0x0C)
-#define EMIF4_0_SDRAM_REF_CTRL          (0x10)
-#define EMIF4_0_SDRAM_REF_CTRL_SHADOW   (0x14)
-#define EMIF4_0_SDRAM_TIM_1             (0x18)
-#define EMIF4_0_SDRAM_TIM_1_SHADOW      (0x1C)
-#define EMIF4_0_SDRAM_TIM_2             (0x20)
-#define EMIF4_0_SDRAM_TIM_2_SHADOW      (0x24)
-#define EMIF4_0_SDRAM_TIM_3             (0x28)
-#define EMIF4_0_SDRAM_TIM_3_SHADOW      (0x2C)
-#define EMIF4_0_SDRAM_MGMT_CTRL         (0x38)
-#define EMIF4_0_SDRAM_MGMT_CTRL_SHD     (0x3C)
-#define EMIF4_0_DDR_PHY_CTRL_1          (0xE4)
-#define EMIF4_0_DDR_PHY_CTRL_1_SHADOW   (0xE8)
-#define EMIF4_0_DDR_PHY_CTRL_2          (0xEC)
-#define EMIF4_0_IODFT_TLGC              (0x60)
+#define EMIF4_0_SDRAM_STATUS		(0x04)
+#define EMIF4_0_SDRAM_CONFIG		(0x08)
+#define EMIF4_0_SDRAM_CONFIG2		(0x0C)
+#define EMIF4_0_SDRAM_REF_CTRL		(0x10)
+#define EMIF4_0_SDRAM_REF_CTRL_SHADOW	(0x14)
+#define EMIF4_0_SDRAM_TIM_1		(0x18)
+#define EMIF4_0_SDRAM_TIM_1_SHADOW	(0x1C)
+#define EMIF4_0_SDRAM_TIM_2		(0x20)
+#define EMIF4_0_SDRAM_TIM_2_SHADOW	(0x24)
+#define EMIF4_0_SDRAM_TIM_3		(0x28)
+#define EMIF4_0_SDRAM_TIM_3_SHADOW	(0x2C)
+#define EMIF4_0_SDRAM_MGMT_CTRL		(0x38)
+#define EMIF4_0_SDRAM_MGMT_CTRL_SHADOW	(0x3C)
+#define EMIF4_0_IODFT_TLGC		(0x60)
+#define EMIF4_0_ZQ_CONFIG		(0xC8)
+#define EMIF4_0_DDR_PHY_CTRL_1		(0xE4)
+#define EMIF4_0_DDR_PHY_CTRL_1_SHADOW	(0xE8)
 
 #define SELF_REFRESH_ENABLE(m)		(0x2 << 8 | (m << 4))
 #define SELF_REFRESH_DISABLE		(0x0 << 8)
-- 
1.7.7.6

